src/kubernetes_api_objects/error.rs

```diff
@@ -16,6 +16,7 @@ pub enum APIError {
     ObjectNotFound,
     ObjectAlreadyExists,
     NotSupported,
+    ServerBusy,
```

Comment:
Can you rename it to `Timeout` or `ServerTimeout`? To align with the official doc: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#response-status-kind